pkg/cmd/roachtest/tests/awsdms.go

```diff
@@ -577,7 +699,7 @@ func setupDMSEndpointsAndTask(
 				Port:               rdsCluster.Port,
 				ServerName:         rdsCluster.Endpoint,
 			},
-			arn: &sourceARN,
+			key: "default_source",
```

Comment:
i think these key names might conflict with `{master, release-22.1, ...}` is running at the same time.

Reply:
It should still generate the name with the suffix of the release at the end though right?
"roachtest-awsdms-rds-endpoint-" + awsdmsVerString(v)"
So there shouldn't be conflict. The key in this code here is mostly just for the map to the ARN of the endpoint

Reply:
didn't realise the key wasn't used for RDS identification. gotcha.